modules/UI/videolayout/FilmStrip.js

```diff
@@ -216,24 +216,24 @@ const FilmStrip = {
 
         return new Promise(resolve => {
             let thumbs = this.getThumbs(!forceUpdate);
-
-            thumbs.localThumb.animate({
-                height: local.thumbHeight,
-                width: local.thumbWidth
-            }, {
-                queue: false,
-                duration: animate ? 500 : 0,
-                complete:  resolve
-            });
-
-            thumbs.remoteThumbs.animate({
-                height: remote.thumbHeight,
-                width: remote.thumbWidth
-            }, {
-                queue: false,
-                duration: animate ? 500 : 0,
-                complete:  resolve
-            });
+            if(thumbs.localThumb)
+                thumbs.localThumb.animate({
+                    height: local.thumbHeight,
+                    width: local.thumbWidth
+                }, {
+                    queue: false,
+                    duration: animate ? 500 : 0,
+                    complete:  resolve
+                });
+            if(thumbs.remoteThumbs)
+                thumbs.remoteThumbs.animate({
+                    height: remote.thumbHeight,
+                    width: remote.thumbWidth
+                }, {
+                    queue: false,
+                    duration: animate ? 500 : 0,
+                    complete:  resolve
+                });
```

Comment:
While the original source code contained a duplication, the two new `if`s are duplications as well. I believe that we should've eliminated the original duplication and added the new `if` in one place by extracting the common source code into a separate function.